boards/nordic/nrf52840_dongle/src/main.rs

```diff
@@ -7,13 +7,15 @@
 // Disable this attribute when documenting, as a workaround for
 // https://github.com/rust-lang/rust/issues/62184.
 #![cfg_attr(not(doc), no_main)]
+#![feature(const_in_array_repeat_expressions)]
 #![deny(missing_docs)]
 
 use capsules::virtual_alarm::VirtualMuxAlarm;
+use kernel::capabilities;
 use kernel::common::dynamic_deferred_call::{DynamicDeferredCall, DynamicDeferredCallClientState};
 use kernel::component::Component;
 #[allow(unused_imports)]
-use kernel::{capabilities, create_capability, debug, debug_gpio, debug_verbose, static_init};
+use kernel::{create_capability, debug, debug_gpio, debug_verbose, static_init};
```

Comment:
```suggestion
use kernel::{capabilities, create_capability, debug, debug_gpio, debug_verbose, static_init};
```
good catch, that was an accident